openhrv/view.py

```diff
@@ -348,7 +349,10 @@ def connect_sensor(self):
         address = self.address_menu.currentText().split(",")[1].strip()
         if not valid_address(address):
             print(f"Invalid sensor address: {address}.")
-            return
+            if platform == "darwin":

```

Comment:
That's interesting, you uncovered a some dead code:
https://github.com/JanCBrammer/OpenHRV/blob/2b0d3ecb72d7be3cf594b1533772b3f0f6ad5715/openhrv/utils.py#L15-L19

The lines above date back to when `OpenHRV` used `bleak` for the sensor connection. On `bleak`, the sensor address format is (used to be?) platform-specific: on Windows and Linux `bleak` uses the MAC format (https://en.wikipedia.org/wiki/MAC_address), whereas on MacOS it uses the UUID format (https://en.wikipedia.org/wiki/Universally_unique_identifier).

`Qt` on the other hand treats all addresses as MACs.

I will remove this guard clause and the platform-specific logic in `utils.py` soon, it's out of scope of this PR.